docs/managing_deis/add_remove_host.rst

```diff
@@ -284,3 +284,28 @@ This can be achieved by making a request to the etcd API. See `remove machines`_
 .. _`remove machines`: https://coreos.com/docs/distributed-configuration/etcd-api/#remove-machines
 .. _`removing monitors`: http://ceph.com/docs/hammer/rados/operations/add-or-rm-mons/#removing-monitors
 .. _`removing OSDs`: http://docs.ceph.com/docs/hammer/rados/operations/add-or-rm-osds/#removing-osds-manual
+
+Automatic Host Removal
+======================
+
+The ``contrib/user-data.example`` provides a unit which provides some experimental logic to clean-up a Deis
+node's Ceph and etcd membership before reboot, shutdown or halt events. Currently, the procedure will
+also be triggered if it is manually stopped via systemctl.
+
+The unit requires that the optional ``deis-store-admin`` component is installed.
+
+.. code-block:: console
+
+    root@deis-1:/# deisctl install deis-store-admin
```

Comment:
this should be `deisctl install store-admin`. The `deis-` is implied by deisctl:

```
><> deisctl install store-admin
deis-store-admin.service: loaded                                 
><> deisctl start store-admin
deis-store-admin.service: activating/start-pre                                 
deis-store-admin.service: active/running
```